test/integration/utility.cc

```diff
@@ -132,6 +145,15 @@ RawConnectionDriver::RawConnectionDriver(uint32_t port, Buffer::Instance& initia
 RawConnectionDriver::~RawConnectionDriver() = default;
 
 void RawConnectionDriver::run(Event::Dispatcher::RunType run_type) { dispatcher_->run(run_type); }
+void RawConnectionDriver::write(absl::string_view payload) {
```

Comment:
newline before and after, same elsewhere.

Reply:
:+1: